samples/CppArExample/CppArExample.cpp

```diff
@@ -80,6 +80,20 @@ void CppArExample::setSceneView(SceneQuickView* sceneView)
   emit sceneViewChanged();
 }
 
+// Update the origin camera using the values send by the calibration view.
+// The m_originCamera is set when the scene is created. The origin camera sets to ArcGISArView is
```

Comment:
Not sure about the third sentence of this comment block. Please review. Thanks.

Reply:
```
// Update the origin camera using the values send by the calibration view.
// The m_originCamera is set when the scene is created. 
// The final origin camera send to ArcGISArView is the sum of m_originCamera 
// and the offset values returned by the calibration view.
```
?